_lib/db.py

```diff
@@ -8,24 +10,33 @@
 
 from .bootstrapping import requires_env
 
+_SQLITE_RE = re.compile(r"^sqlite:///(.*)$")
+
 
 @click.group()
 def db():
     pass
 
 
-@db.command()
-@requires_env("app")
-def ensure():
+def _create_sqlite():
+    from flask_app.models import db
+    from flask_app.app import app
+    uri = app.config['SQLALCHEMY_DATABASE_URI']
+    path = _SQLITE_RE.search(uri).group(1)
+
+    if os.path.exists(path):
+        logbook.info("{} exists. Not doing anything", path)
```

Comment:
Like in postgres, we don't want to call create_all if the database already exists. Since it's Sqlite, the database exists of the file exists